disnake/http.py

```diff
@@ -101,6 +102,16 @@ def _workaround_set_api_version(version: Literal[9, 10]) -> None:
     Route.BASE = f"https://discord.com/api/v{_API_VERSION}"
 
 
+def get_user_agent() -> str:
+    user_agent = (
+        "DiscordBot (https://github.com/DisnakeDev/disnake {0}) Python/{1[0]}.{1[1]} aiohttp/{2}"
+    )
+    return user_agent.format(__version__, sys.version_info, aiohttp.__version__)
```

Comment:
this could be an f-string

Reply:
could be but I'm not sure I like it, https://github.com/DisnakeDev/disnake/pull/959/commits/bfaddb3d1ce2bf464f724a650aa73c972d3e3197

Reply:
reverted in [`dd15dc6` (#959)](https://github.com/DisnakeDev/disnake/pull/959/commits/dd15dc649a080fc439879e6fa61776f125c31eea)